example-get-started/generate.sh

```diff
@@ -26,7 +26,9 @@ export VIRTUAL_ENV_DISABLE_PROMPT=true
 source .venv/bin/activate
 echo '.venv/' > .gitignore
 
-pip install "dvc[s3]"
+
+pip install gitpython
+pip install "git+https://github.com/iterative/dvc#egg=dvc[all]"
```

Comment:
This is temporary though, right? Should just be `pip install dvc` once 1.0 is fully released.

Reply:
it's a good question :) I would say it's even more useful for a us to keep it to the latest master?

Reply:
OK yeah maybe here in the gen, but not in the actual Get Started 🙂 

Also, only `git+https://github.com/iterative/dvc#egg=dvc[s3]` (S3) is needed here, I think.

Reply:
oh, yeah. Not in the actual get started, that's for sure. Even here - I don't have a strong opinion to be honest. It feels it can be useful since we'll need it to test new iterations down the road.